examples/plot_simulate_gamma.py

```diff
@@ -108,6 +108,22 @@
 from hnn_core.viz import plot_psd
 plot_psd(dpls[trial_idx], fmin=20., fmax=100., tmin=tmin)
 
+###############################################################################
+# Finally, we demonstrate the mechanistic link between PING and the GABAA decay
+# time constant (`tau2`). Using the same network/drive configuration as before,
+# we decrease `tau2` from 5 to 2 ms. This will shorten the effective
+# refactory period between L5 pyramidal cell spikes and increase the PING
+# frequency from ~50 to ~65 Hz.
+net.cell_types['L5_pyramidal'].p_syn['gabaa']['tau2'] = 2
+net.update_cells()
```

Comment:
Here's what the API looks like for changing a biophysical synapse parameter. Note that it also produces the anticipated results before vs. after manual adjustment of `tau2`:
![Screenshot from 2021-05-14 15-02-48](https://user-images.githubusercontent.com/20212206/118320346-1e51c180-b4ca-11eb-8f93-a8c89fade65a.png)

Reply:
yeaahhhh! this is the right direction to go :) Could we hide `update_cells` from the user? They shouldn't have to worry about it. Should be called during the build somewhere

Reply:
Oh good point, I think I can throw it in `NetworkBuilder._create_cells_and_drives()`.